src/distilabel/steps/keep.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2023-present, Argilla, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Iterator, List
+
+from distilabel.steps.base import Step, StepInput
+
+
+class KeepColumns(Step):
```

Comment:
Should we keep this `Step`? I included it because it was really useful to me, but wanted to know your opinion @gabrielmbmb @plaguss before adding the docstrings

Reply:
yes